kyo-tapir/shared/src/main/scala/kyo/routes.scala

```diff
@@ -30,8 +30,10 @@ object Routes:
     ): Unit < Routes =
         Sums.add(
             Route(
-                e.serverSecurityLogic[A, KyoSttpMonad.M](a => Right(a)).serverLogic(a =>
```

Comment:
Did you have type inference issues here?

Reply:
I did! Maybe that's the the Nothing >: thing! Or something else I did.

Reply:
Let me try to revert the Nothing and see 😄